manticore/core/manticore.py

```diff
@@ -1065,18 +1082,33 @@ def run(self):
         # different process modifies the state
         self.stcache = weakref.WeakValueDictionary()
 
+        if self._introspector is None:
+            self._introspector = self._introspection_plugin()
+            self.register_plugin(self._introspector)
```

Comment:
That's fine with me. It doesn't really make sense to run the introspector before `run` is called (since there aren't any states), but I don't think it should break anything.

Reply:
So as I understand this the plugins in general will only _run_ from receiving events inside a manticore.run(). An then they can be configured or queried from the script main.